plugins/auth-basic/src/index.ts

```diff
@@ -1,13 +1,12 @@
-import { resolve } from "path";
 import {
-  DsgContext,
-  CreateServerAuthParams,
   AmplicationPlugin,
-  Events,
   CreateAdminUIParams,
+  CreateServerAuthParams,
+  DsgContext,
+  Events,
 } from "@amplication/code-gen-types";
 import { EnumAuthProviderType } from "@amplication/code-gen-types/src/models";
-
+import { staticsPath } from "./constants";
```

Comment:
please align this static path and implement it like we are doing with the other packages. We don't need a constant file to hold the path to the static files